dfinity/oisy-wallet
# feat(frontend): auto-enable ckERC20 tokens when converting from ERC20

# Motivation

Same as PR #2292 , when the user converts ERC20 to ckERC20, it is useful that the ckERC20 token is enabled if it is not. We use the same flow as the one used for `ckERC20 --> ERC20`: when the user click the Convert button if the token is not enabled, it will be.

# Tests

In local replica, the following test worked:

- Disable ckSepoliaUSD.
- Convert Sepolia USDC to ckSepoliaUSD.
- Check if ckSepoliaUSD was enabled.


https://github.com/user-attachments/assets/fbf58751-2533-4f8b-b6e6-22fdbaa5a055
